### PR TITLE
add support for filtering integrations by description

### PR DESCRIPTION
**Pull request by philwhiteuk**
requires additional `RUNSCOPE_INTEGRATION_DESC` env to be provided

### CODE REVIEW

File: plugin/providers/runscope/data_source_runscope_integration.go [hunk 1]
```diff
@@ -16,10 +16,32 @@ func dataSourceRunscopeIntegration() *schema.Resource {
...
+						},
+						"values": {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
 			"type": {
```
Comment (ewilde) [inline plugin/providers/runscope/data_source_runscope_integration.go:37]: now that we have filters, do you think we should get rid of type as a top level schema attribute?

Comment (ewilde) [reply plugin/providers/runscope/data_source_runscope_integration.go:37]: looks great :smile:  btw!

Comment (philwhiteuk) [reply plugin/providers/runscope/data_source_runscope_integration.go:37]: I think using filter is more intuitive, but would not be backwards compatible so left `type` in to support both

Comment (ewilde) [reply plugin/providers/runscope/data_source_runscope_integration.go:37]: makes sense, thanks for the PR

**Pull request merged**